src/component/TimeBox/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledTime = styled.div`
+  color : #6c757d;
+  font-size : 13px;
+  border : 1px solid white;
+  text-align : center;
+  align-items : center;
+  margin-left : auto;
+  display: flex;
+  flex-direction : column;
+  justify-content : center;
+`;
+
+const TimeBox =(props) =>{
+  return(
+  <StyledTime>{props.children}</StyledTime>
+  );
+};
+
+export default TimeBox;
```

Comment:
```suggestion
const TimeBox = styled.div`
  color : #6c757d;
  font-size : 13px;
  border : 1px solid white;
  text-align : center;
  align-items : center;
  margin-left : auto;
  display: flex;
  flex-direction : column;
  justify-content : center;
`;

export default TimeBox;
```

`TimeBox` 컴포넌트는 단순히 `StyledTime` 에 props 를 그대로 전달해주는 역할만하고 있어요. 위와 같이 코드를 개선해볼 수 있을 것 같아요 😁